src/Uno.UWP/System/Launcher.Android.SpecialUris.cs

```diff
@@ -0,0 +1,95 @@
+﻿#if __ANDROID__
+using System.Linq;
+using System.Threading.Tasks;
+using Android.Content;
+using System;
+using Android.Provider;
+using System.Collections.Generic;
+
+namespace Windows.System
+{
+	public static partial class Launcher
+	{
+		private const string MicrosoftSettingsUri = "ms-settings";
+
+		private static readonly Dictionary<string, string> _settingsHandlers = new Dictionary<string, string>() {
+			{"sync", Settings.ActionSyncSettings},
+			{"appsfeatures-app", Settings.ActionApplicationDetailsSettings},
+			{"appsfeatures", Settings.ActionApplicationSettings},
+			{"defaultapps", Settings.ActionManageDefaultAppsSettings},
+			{"appsforwebsites", Settings.ActionManageDefaultAppsSettings},
+			{"cortana", Settings.ActionVoiceInputSettings},
+			{"bluetooth", Settings.ActionBluetoothSettings},
+			{"printers", Settings.ActionPrintSettings},
+			{"typing", Settings.ActionHardKeyboardSettings},
+			{"easeofaccess", Settings.ActionAccessibilitySettings},
+			{"network-airplanemode", Settings.ActionAirplaneModeSettings},
+			{"network-celluar", Settings.ActionNetworkOperatorSettings},
+#if __ANDROID_28__
```

Comment:
this define is when the app is built for android 9 but do we handle the app not running on android28 ? What happens in that case ?

Reply:
Fixed, made this `Lazy`, so that this particular link is added only if running on a supported OS. Previous versions will just fallback to `network`, which gives the networking settings main page 😉